File: src/main/java/plugin/rsinterface/JournalInterfacePlugin.java
package plugin.rsinterface;

import org.redrune.game.content.plugin.type.InterfacePlugin;
import org.redrune.game.entity.actor.player.Player;

/**
 * @author Tyluur
 * @since 2019-05-22
 */
public class JournalInterfacePlugin implements InterfacePlugin {
	
	@Override
	public boolean handle(Player player, int interfaceId, int componentId, int itemId, int slotId, int packetId) {
		return false;
	}
	
	@Override
	public void register() {
	
	}
}

File: src/main/kotlin/plugin/rsinterface/JournalInterfacePlugin.kt
package plugin.rsinterface

import org.redrune.game.content.plugin.type.InterfacePlugin
import org.redrune.game.entity.actor.player.Player

/**
 * @author Tyluur
 * @since 2019-05-22
 */
class JournalInterfacePlugin : InterfacePlugin {
    override fun handle(
        player: Player,
        interfaceId: Int,
        componentId: Int,
        itemId: Int,
        slotId: Int,
        packetId: Int,
    ): Boolean {
        return false
    }

    override fun register() {}
}